client/wfnews-war/src/main/angular/src/app/components/report-of-fire/review-page/rof-review-page.component.ts

```diff
@@ -194,9 +201,86 @@ export class RoFReviewPage extends RoFPage implements AfterViewInit{
         const label = button.label;
         return label
       }
-      else {
-        return "I\'m not sure"
+    }
+  }
+
+  async submitReport() {
+    if (!this.checkOnlineStatus) {
+      await this.storage.set('offlineReportData', this.reportOfFire);
+    }
+  }
+
+  ionViewDidEnter() {
+    this.scheduleDataSync();
+  }
+
+  async scheduleDataSync() {
+    await this.storage.create();
+    const syncIntervalMinutes = 0.1;
+    setInterval(async() => {
+      const isConnected = await this.checkOnlineStatus();
+      console.log('isConnected:', isConnected);
+      if (isConnected) {
+        console.log('Syncing data with the server');
+        await this.syncDataWithServer();
+      }
+    }, syncIntervalMinutes *60 * 1000);
```

Comment:
Might want to make this a configuration as well, so we can adjust the sync timeout